ddp-workspace/projects/ddp-sdk/src/lib/directives/addressGoogleAutocomplete.directive.ts

```diff
@@ -132,4 +146,20 @@ export class AddressGoogleAutocompleteDirective implements OnInit, OnDestroy, On
         const keys = _.keys(place);
         return keys.length === 1 && keys[0] === 'name';
     }
+
+    private getPotentiallyFixedStreetAddress(route: string): string {
+        const selectedAddress = this.autocompleteInput.nativeElement.value;
+        let selectedAddressHouseNumber = selectedAddress.split(' ')[0];
+
+        // Check if first element is a number
+        if (isNaN(+selectedAddressHouseNumber)) {
```

Comment:
You know, this check makes a lot of sense EXCEPT that I believe in some cases, "house numbers" can include letters, e.g.:
https://www.google.com/maps/place/500A+E+87th+St,+New+York,+NY+10128/@40.7765337,-73.9464339,3a,75y,197.63h,90t/data=!3m7!1e1!3m5!1szyk2YT-cV2M8f0PEqQ5bCA!2e0!6shttps:%2F%2Fstreetviewpixels-pa.googleapis.com%2Fv1%2Fthumbnail%3Fpanoid%3Dzyk2YT-cV2M8f0PEqQ5bCA%26cb_client%3Dsearch.gws-prod.gps%26w%3D86%26h%3D86%26yaw%3D197.62915%26pitch%3D0%26thumbfov%3D100!7i16384!8i8192!4m5!3m4!1s0x89c258b08d691b13:0xc1745a8608b1d122!8m2!3d40.776326!4d-73.9465025

Though this might be the best you can do here.

Reply:
Just a question: what does the + operator do here? coerce to a number?

Reply:
@MocanaAtBroad correct! Just convert a string (or any other type) to a number.